WooCommerce/Classes/ViewModels/Order Details/Refunded Products/RefundedProductsDataSource.swift

```diff
@@ -139,7 +140,7 @@ extension RefundedProductsDataSource {
     ///
     func reloadSections() {
         let refundedProducts: Section? = {
-            let rows: [Row] = Array(repeating: .orderItemRefunded, count: items.count)
+            let rows: [Row] = Array(repeating: .orderItemRefunded, count: self.refundedProducts.count)
```

Comment:
nit: `self` might not be necessary?

Reply:
Poor naming on my part. The `refundedProducts` section is named the same as the `refundedProducts` class member. Fixed ✅